pkg/cmd/kind/export/logs/logs.go

```diff
@@ -80,12 +80,11 @@ func runE(logger log.Logger, streams cmd.IOStreams, flags *flagpole, args []stri
 		dir = args[0]
 	}
 
-	// collect the logs
-	if err := provider.CollectLogs(flags.Name, dir); err != nil {
-		return err
-	}
-
-	logger.V(0).Infof("Exported logs for cluster %q to:", flags.Name)
+	// NOTE: the path is the output of this command to be captured by calling tools
+	// whereas "Exporting logs..." is info / debug (stderr)
+	logger.V(0).Infof("Exporting logs for cluster %q to:", flags.Name)
```

Comment:
to: %s", dir)  ??

Reply:
no, we print the path to stdout for capture by the calling program.
we print the message to stderr.
(see the comment above)

Reply:
e.g.:
```console
$ kind export logs
Exporting logs for cluster "kind" to: 
/private/var/folders/rt/xdpktcmj0p7fx3kvwmrgdqyh00fsfv/T/609940926
ERROR: [command "docker exec --privileged kind-control-plane sh -c 'tar --hard-dereference -C /var/log/ -chf - . || (r=$?; [ $r -eq 1 ] || exit $r)'" failed with error: exit status 1, [command "docker exec --privileged kind-control-plane cat /kind/version" failed with error: exit status 1, command "docker exec --privileged kind-control-plane journalctl --no-pager -u kubelet.service" failed with error: exit status 1, command "docker exec --privileged kind-control-plane journalctl --no-pager -u containerd.service" failed with error: exit status 1, command "docker exec --privileged kind-control-plane journalctl --no-pager" failed with error: exit status 1]]
```

the stdout part is `/private/var/folders/rt/xdpktcmj0p7fx3kvwmrgdqyh00fsfv/T/609940926`

Reply:
note: in this commit I basically just moved this up before we actually export (so if we hit an error you can still see where the logs we did collect are) and s/Exported/Exporting/